data/tx/legislature/Alex-Dominguez-957c21d3-801a-45ee-b3ef-ee1e70624306.yml

```diff
@@ -12,12 +12,8 @@ roles:
   district: '37'
 offices:
```

Comment:
@sroomf Move to retired, this person actually left on 01/10/2023. This seat House District 37 is now held by Janie Lopez - https://ballotpedia.org/Janie_Lopez - but I don't see a file for them in this pr.